src/main/webapp/app/exercises/programming/shared/commits-info/commits-info-group/commits-info-row/commits-info-row.component.html

```diff
@@ -0,0 +1,74 @@
+<tr class="{{ isExpanded ? (groupCommitIndex === groupCommitCount - 1 ? 'bg-light border-bottom' : 'bg-light') : '' }}">
+    <th scope="row" class="p-2">
+        @if (groupCommitIndex === 0) {
+            {{ pushNumber }}
+        } @else {
+            <span class="text-body-secondary">
+                {{ pushNumber + '.' + (groupCommitIndex + 1) }}
+            </span>
+        }
+    </th>
+    <td class="p-2">
+        @if (localVC) {
+            @if (isRepositoryView) {
+                <a [routerLink]="['./', commit.hash]" ngbTooltip="{{ commit.hash }}"> {{ commit.hash ?? '' | truncate: 7 }} </a>
+            } @else {
+                <span ngbTooltip="{{ commit.hash }}">
+                    {{ commit.hash ?? '' | truncate: 7 }}
+                </span>
+            }
+        } @else {
+            <a [href]="commit.commitUrl" ngbTooltip="{{ commit.hash }}"> {{ commit.hash ?? '' | truncate: 7 }} </a>
+        }
+    </td>
+    <td class="p-2">
+        @if (isRepositoryView) {
+            {{ user?.name ?? commit.author }}, {{ user?.email ?? commit.authorEmail }}
+        } @else {
+            {{ user?.name }}
+        }
+    </td>
+    <td class="p-2">
+        {{ commit.timestamp | artemisDate: 'long' : true }}
+    </td>
+    @if (isRepositoryView) {
+        <td class="p-2">
+            @if (commit.result) {
+                <jhi-result [result]="commit.result" [showUngradedResults]="true" [showBadge]="true" />
+            } @else if (groupCommitIndex === 0) {
+                <span class="text-body-secondary">
+                    <fa-icon [icon]="faCircle" size="lg" />&nbsp;
+                    <span jhiTranslate="artemisApp.result.noResult"></span>
+                </span>
+            }
```

Comment:
`jhi-result` should already handle the case when there is no result

Reply:
Yes, the jhi-result handles the noResult case. I took this logic from the original commits-info code. My guess is that the intention behind having a custom noResult UI is to display the circle. Maybe jhi-result could be updated to optionally allow displaying icons to the left of the "No Result" text.

#### Current implementation:


<img width="348" alt="image" src="https://github.com/user-attachments/assets/cd0452aa-1ed7-4c64-ac72-72cf14292e51">

#### Implementation when using the default noResult UI:

<img width="319" alt="image" src="https://github.com/user-attachments/assets/a6fbbdef-4471-4c9c-b784-588fb3184dfd">